<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmCompatibilidadExe</class>
 <widget class="QDialog" name="frmCompatibilidadExe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>390</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Compatibilidad</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="gBox_compatibilidad_modo">
     <property name="title">
      <string>Ejecutar este programa en el modo de compatibilidad para:</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="hLayout_compatibilidad_modo" stretch="0,1">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="lb_Datos_43">
        <property name="text">
         <string>Compatibilidad</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbxDat_compatibilidad_modo">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gBox_compatibilidad_pantalla">
     <property name="title">
      <string>Configuración</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QCheckBox" name="chkDat_compatibilidad_color_reducido">
        <property name="text">
         <string>Modo de color reducido</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbxDat_compatibilidad_color_reducido">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkDat_compatibilidad_resolucion_640x480">
        <property name="text">
         <string>Ejecutar con una resolución de pantalla de 640 x 480</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkDat_compatibilidad_deshabilitar_temas">
        <property name="text">
         <string>Deshabilitar los temas visuales</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkDat_compatibilidad_ejecutar_admin">
        <property name="text">
         <string>Ejecutar este programa como administrador</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gBox_compatibilidad_entrada">
     <property name="title">
      <string>Configuración de entrada</string>
     </property>
     <layout class="QVBoxLayout" name="vLayout_compatibilidad_entrada">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QCheckBox" name="chkDat_compatibilidad_desactivar_servicios_avanzados_texto">
        <property name="text">
         <string>Desactivar servicios avanzados de texto para este programa</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="minimumSize">
        <size>
         <width>97</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Aceptar</string>
       </property>
       <property name="icon">
        <iconset resource="../res/gr-lida.qrc">
         <normaloff>:/img16/aplicar.png</normaloff>:/img16/aplicar.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="minimumSize">
        <size>
         <width>97</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
       <property name="icon">
        <iconset resource="../res/gr-lida.qrc">
         <normaloff>:/img16/cancelar.png</normaloff>:/img16/cancelar.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/gr-lida.qrc"/>
 </resources>
 <connections/>
</ui>
